presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -2173,7 +2173,9 @@ public PhysicalOperation visitTableWriter(TableWriterNode node, LocalExecutionPl
                         new DataSize(0, BYTE),
                         context,
                         2,
-                        outputMapping);
+                        outputMapping,
+                        200,
```

Comment:
@arhimondr Assuming that this number ties to max number of output partitions, let's make the connection explicit. E.g. add a comment or, better yet, use the same configuration setting to compute this.

Reply:
I don't think it is worth configuration. It is not a hard limit, but the initial size for the grouping hashmap. It will grow larger if needed.